src/hooks/screen/useSignedChannelListHook.ts

```diff
@@ -9,7 +9,8 @@ import {getUserId} from '../../utils/token';
 
 function useSignedChannelListScreenHook(): UseSignedChannelListScreenHook {
   const {localDb, channelList} = useLocalDatabaseHook();
-  const {goToChatScreen, goToPostDetailScreen, goToCommunityScreen} = useChatUtilsHook();
+  const {goToChatScreen, goToPostDetailScreen, goToCommunityScreen, goToContactScreen} =
+    useChatUtilsHook();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `goToContactScreen` function from `useChatUtilsHook()` is a good improvement. It seems like it will provide a way to navigate to the contact screen, which could be useful in this context. However, please ensure that this function is properly defined and tested in `useChatUtilsHook()`.
```diff
+   const {goToChatScreen, goToPostDetailScreen, goToCommunityScreen, goToContactScreen} =
+     useChatUtilsHook();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->